go/cmd/pathwar/api.go

```diff
@@ -157,10 +159,15 @@ func apiCommand() *ffcli.Command {
 }
 
 func svcFromFlags(logger *zap.Logger) (pwapi.Service, *gorm.DB, func(), error) {
+	// logger
+	gormLogger := zapgorm2.New(zap.L())
+	gormLogger.SetAsDefault()
+	pwdb.DefaultGormConfig.Logger = gormLogger
+
```

Comment:
Doesn't print any log

Reply:
I can give it a look, but I suggest you to check if that code is called before your `gorm.Open` and not after?

Reply:
According to the doc (https://gorm.io/docs/logger.html) it seems to be at the right place if we want it globally